Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/LeapMotionConfigurationChecker.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion
+{
+    /// <summary>
+    /// Class that checks if the Leap Motion Core assets are present and configures the project if they are.
+    /// </summary>
+    [InitializeOnLoad]
+    static class LeapMotionConfigurationChecker
+    {
+        private const string FileName = "README_BEFORE_UPDATING.txt";
```

Comment:
Can we have a couple of changes here:
1. A slightly more descriptive name than "FileName"
1. A comment explaining what this is for (i.e. used to detect if Leap exists)

The tricky part is that this is still a somewhat generically named thing as well - I don't know if we want to do a secondary check to filter things down a bit more.

Reply:
I originally had the LeapXRServiceProvider.cs as the detection file, but I changed the file to the Read Me as an attempt at creating a Leap Motion Remover menu item.  I used the read me because the removal of a txt file did not result in errors. I can change it back to a more leap specific file.

Reply:
Added comment and more descriptive name